src/Import/Data/Normalizer.php

```diff
@@ -102,6 +103,13 @@ private function compileObjectProperties(Immobilie $immobilie): array
                 'Anlage' => $nutzungsart?->getAnlage() ?? false,
                 'WAZ' => $nutzungsart?->getWaz() ?? false,
             ]),
+            // Vermarkungsart
+            'vermarktungsart' => $this->serializeFlags([
+                'Kauf' => $nutzungsart?->getKauf() ?? false,
+                'Miete' => $nutzungsart?->getMiete() ?? false,
+                'Pacht' => $nutzungsart?->getPacht() ?? false,
+                'Erbpacht' => $nutzungsart?->getErbpacht() ?? false,
```

Comment:
This doesn't look right. Did you mean to use `$vermarktungsart`? But then these would be the properties that are available in `Ujamii\OpenImmo\API\Vermarktungsart`:

```suggestion
                'Kauf' => $vermarktungsart?->getKauf() ?? false,
                'Miete' => $vermarktungsart?->getMietePacht() ?? false,
                'Pacht' => $vermarktungsart?->getLeasing() ?? false,
                'Erbpacht' => $vermarktungsart?->getErbpacht() ?? false,
```